examples/Notebook/Converting Notebooks With nbconvert.ipynb

```diff
@@ -50,230 +46,105 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We will be converting `Custom Display Logic.ipynb`. \n",
-    "Be sure to have runed some of the cells in it to have output otherwise you will only see input in nbconvert.\n",
-    "Nbconvert **do not execute the code** in the notebook files, it only converts what is inside."
+    "As a test, the `Index.ipynb` notebook in the directory will be convert. \n",
+    "\n",
+    "If you're converting a notebook with code in it, make sure to run the code cells that you're interested in before attempting to convert the notebook.  Nbconvert **does not execute the code cell** of the notebooks that it converts."
```

Comment:
Oh that's right, I forgot about that. - But it doesn't without being requested.  I'll change the wording so it's not as aggressive.